packages/social-trader/src/components/details/details-statistic-section/details.chart.helpers.ts

```diff
@@ -4,11 +4,11 @@ import {
   BalanceChartPoint,
   FundBalanceChart,
   FundChartStatistic,
-  FundProfitPercentCharts,
+  FundProfitCharts,
```

Comment:
Revert this file. Correct interfaces is `ProgramProfitPercentCharts` , `FundProfitPercentCharts `